test/fakeintake/aggregator/containerAggregator.go

```diff
@@ -44,6 +44,9 @@ func ParseContainerPayload(payload api.Payload) ([]*ContainerPayload, error) {
 	switch m := msg.Body.(type) {
 	case *agentmodel.CollectorContainer:
 		return []*ContainerPayload{{CollectorContainer: *m, collectedTime: payload.Timestamp}}, nil
+	case *agentmodel.CollectorContainerRealTime:
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution. -->
**❓ question**
Is the aggregator currently failing because the payload is unexpected?

Reply:
Yes, this was causing the flakiness issue we were seeing due to the dual shipping in the k8s environment.

Reply:
Ah! That is indeed a bummer! Ok for the fix!

Reply:
@wiyu Could you link the Jira card for RT mode coverage here also?